components/nimbus/src/targeting.rs

```diff
@@ -61,6 +48,16 @@ impl NimbusTargetingHelper {
         }
     }
 
+    pub fn evaluate_jexl_raw_value(&self, expr: String) -> Result<Value> {
```

Comment:
This doesn't need to take a String. You're immediately taking a ref to pass it to jexl_eval_raw. It should take a &str

```suggestion
    pub fn evaluate_jexl_raw_value(&self, expr: &str) -> Result<Value> {
```